eng/pipelines/runtime.yml

```diff
@@ -151,15 +151,15 @@ jobs:
 #
 # Build PGO CoreCLR release
 #
-#- template: /eng/pipelines/common/platform-matrix.yml
-#  parameters:
-#    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
-#    buildConfig: release
-#    platforms:
-#    - windows_x64
-#    jobParameters:
-#      testGroup: innerloop
-#      pgoType: 'PGO'
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
+    buildConfig: release
+    platforms:
+    - windows_x64
+    jobParameters:
+      testGroup: innerloop
+      pgoType: 'PGO'
```

Comment:
do we expect different PGO types or why didn't you choose a bool?

Reply:
I couldn't figure out a good way in YAML to turn a bool into a string, and didn't want to repeat a conditional in the build where I change the name of jobs to include "PGO". If you have a better way I'd love to use it. I couldn't find any support for something like `if(pgoInstrument, 'PGO', '')` in YAML.

Reply:
ah I see. Unfortunately there is no way in YAML to do that. So let's keep it as is.